src/app/header/header.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit } from "@angular/core";
+import { MatIconRegistry } from "@angular/material/icon";
+import { DomSanitizer } from "@angular/platform-browser";
+
+@Component({
+  selector: "app-header",
+  templateUrl: "./header.component.html",
+  styleUrls: ["./header.component.css"],
+})
+export class HeaderComponent implements OnInit {
+  constructor(
+    private matIconRegistry: MatIconRegistry,
+    private domSanitizer: DomSanitizer
+  ) {
+    this.matIconRegistry.addSvgIcon(
+      "user",
+      this.domSanitizer.bypassSecurityTrustResourceUrl(
+        "../../assets/icon/user-records.icon.svg"
```

Comment:
Are `.svg` files included in Git LFS?

Reply:
It was initially not included, but is included now.